### PR TITLE
base.html: Add footer

### PR DESCRIPTION
**Pull request by sagar-kalra**
Add footer in base.html with links to social sites.

Closes https://github.com/pclubuiet/website/issues/3

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -14,6 +14,20 @@
 
     <!-- Optional theme -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
+</head>
+<style>
+.footer {
```
Comment (divyam3897) [inline templates/base.html]: Add all the styles to the seperate .CSS file (base.css if it's a general style) else app specific .css

**Comment by sagar-kalra** [comment]
@dmadaan Done.

File: templates/base.html [hunk 2]
```diff
@@ -33,6 +36,17 @@
 </nav>
 <div id = "content">
 {% block content %}{% endblock %}
+</div>
+<footer class='footer' style="background-color:#222">
```
Comment (divyam3897) [inline templates/base.html]: why two colors for footer? Even one in external css

File: templates/base.html [hunk 3]
```diff
@@ -33,6 +36,17 @@
...
+<br>
+  <div class="container text-center">
+    <a href="bit.ly/pclubfb"><i class="fa fa-facebook fa-3x"></i></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (divyam3897) [inline templates/base.html]: I land at http://127.0.0.1:8000/home/bit.ly/pclubfb

File: templates/base.html [hunk 4]
```diff
@@ -33,6 +36,17 @@
...
+    <a href="tinyurl.com/pclubslack"><i class="fa fa-slack fa-3x"></i></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (divyam3897) [inline templates/base.html]: similarly for http://127.0.0.1:8000/home/tinyurl.com/pclubslack

File: templates/base.html [hunk 5]
```diff
@@ -14,7 +14,10 @@
 
     <!-- Optional theme -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
-
+    <link rel = 'stylesheet' href="{% static 'home/base.css' %}">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
```
Comment (divyam3897) [inline templates/base.html]: why these changes? They were already present.

File: templates/base.html (hunk 2 above)
Comment (divyam3897) [reply templates/base.html]: keep the external one and not here.

File: templates/base.html (hunk 5 above)
Comment (sagar-kalra) [reply templates/base.html]: i changed it, added https:// before the links
Thats why they are working now.

Comment (sagar-kalra) [reply templates/base.html]: there was an extra line, i just removed it that made the code to shift upwards

**Comment by divyam3897** [comment]
It has been merged, thanks :)
